stacks-common/src/types/mod.rs

```diff
@@ -431,7 +432,7 @@ impl SIP031EmissionInterval {
 
 impl StacksEpochId {
     pub fn latest() -> StacksEpochId {
-        StacksEpochId::Epoch32
+        StacksEpochId::Epoch33
```

Comment:
I'm not 100% sure that the same logic about the peer version / miner markers applies here, but I think it might: I'm not convinced that the stacks node should consider 3.3 the latest epoch version until the node actually could run 3.3.

Reply:
Changed in c6074fb98ab4346fdde5354b89662b227bc779c9.